doc/conf.py

```diff
@@ -56,6 +56,11 @@
 
 hawkmoth_root = os.path.join(os.path.abspath(os.path.dirname(__file__)), '../test/examples')
 
+source_uri = 'https://github.com/jnikula/hawkmoth/tree/{version}/test/examples/{{source}}#L{{line}}'
+source_version = version if len(version.split('.')) == 3 else 'master'
```

Comment:
Wondering if we should have an intermediate fallback to a specific hash revision instead of using a branch name which is moving target. But I guess it's fine for the current use case.

Reply:
That requires looking at git, and opens up more problems. I'd revisit that later.

I did notice the VERSION file does not have v prefix, but the tags do, so I fixed that.